packages/syft/src/syft/service/worker/utils.py

```diff
@@ -16,16 +24,100 @@
 from .worker_pool import WorkerStatus
 
 
-def run_container_using_docker(
+def docker_build(
+    image: SyftWorkerImage
+) -> Union[Tuple[Image, Iterable[str]], SyftError]:
+    try:
+        builder = CustomWorkerBuilder()
+        (image, logs) = builder.build_image(
+            config=image.config,
+            tag=image.image_tag.full_tag,
+        )
+        parsed_logs = parse_output(logs)
+        return (image, parsed_logs)
+    except docker.errors.APIError as e:
+        return SyftError(
+            message=f"Docker API error when building {image.image_tag}. Reason - {e}"
+        )
+    except docker.errors.DockerException as e:
+        return SyftError(
+            message=f"Docker exception when building {image.image_tag}. Reason - {e}"
+        )
+    except Exception as e:
+        return SyftError(
+            message=f"Unknown exception when building {image.image_tag}. Reason - {e}"
+        )
+
+
+def docker_push(
+    self,
+    image: SyftWorkerImage,
+    username: str = "",
+    password: str = "",
```

Comment:
Should just be

```suggestion
    username: str,
    password: str,
```

for now. We should not hard code default username/pw here. Plus it's failing snyk `B107:hardcoded_password_default` check. We can change these to `Optional[str]` later if needed.